From ed60d4605b81c43b9ba9504a37835109c247c6f8 Mon Sep 17 00:00:00 2001
From: Stefan Schmidt <stefan.schmidt@huawei.com>
Date: Fri, 1 Apr 2022 21:46:03 +0200
Subject: [PATCH] otbr-agent.service.in: remove pre exec hook for mdns service

It uses the service command which is not available in all cases under
Yocto/OE. The upstream project uses this mainly with Ubuntu and Raspian
as testbeds.

In our case we simply ensure that avahi-daemon is installed on the
system inside the recipe.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Stefan Schmidt <stefan.schmidt@huawei.com>
---
 src/agent/otbr-agent.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/agent/otbr-agent.service.in b/src/agent/otbr-agent.service.in
index 8314121347..4c97869def 100644
--- a/src/agent/otbr-agent.service.in
+++ b/src/agent/otbr-agent.service.in
@@ -6,7 +6,7 @@ After=dbus.socket
 
 [Service]
 EnvironmentFile=-@CMAKE_INSTALL_FULL_SYSCONFDIR@/default/otbr-agent
-@EXEC_START_PRE@ExecStart=@CMAKE_INSTALL_FULL_SBINDIR@/otbr-agent $OTBR_AGENT_OPTS
+ExecStart=@CMAKE_INSTALL_FULL_SBINDIR@/otbr-agent $OTBR_AGENT_OPTS
 KillMode=mixed
 Restart=on-failure
 RestartSec=5
-- 
2.35.1

